examples/android_local_test/MODULE.bazel.lock

```diff
@@ -0,0 +1,1659 @@
+{
```

Comment:
Please remove this file from the PR. Right now, the `MODULE.bazel.lock` files are notoriously unstable when used cross-platform, and it's best to leave them out of a repo which may be used on different OSs